android/src/main/java/com/reactnativesimplebiometrics/SimpleBiometricsModule.java

```diff
@@ -96,4 +95,22 @@ public void onAuthenticationSucceeded(@NonNull BiometricPrompt.AuthenticationRes
         );
 
     }
+
+    @ReactMethod
+    public void getBiometryType(Promise promise) {
+        try {
+            ReactApplicationContext context = getReactApplicationContext();
+            PackageManager pm = context.getPackageManager();
+
+            if (pm.hasSystemFeature(PackageManager.FEATURE_FINGERPRINT)) {
+                promise.resolve("Fingerprint");
+            } else if (pm.hasSystemFeature(PackageManager.FEATURE_FACE)) {
```

Comment:
can you add some references to docs explaining how this works?

does this handles cases where the system has both features but the user has only enabled one of them?